### DISCUSSION TITLE
2023-11-05 puzzle solvable?

### DISCUSSION THREAD

**Discussion by graue** [gereleth/hexapipes#104] [General]
The 2023-11-05 puzzle seems impossible to me. There's a relatively small area where every possibility I can come up with leads to a contradiction. I restarted several times, always get stuck here and don't see my error.

[Screencast from 11-04-2023 11:10:25 PM.webm](https://github.com/gereleth/hexapipes/assets/1730853/2cf4d4b8-8eb5-46c0-b77a-21d9d469377b)

**Comment by Jaybrainer** [comment]
Hello graue,

I verified that the puzzle is solvable by hand after seeing your post. I can say that you have made a mistake in the first 15 seconds of the video.

**Reply by graue** [reply]
Ah. Somehow I failed to notice that there were 2 different ways for the two adjacent ![image](https://github.com/gereleth/hexapipes/assets/1730853/307063c7-60c6-4946-ab8c-d5309b1b5f6c)s to touch.

Well now I'm embarrassed, but also impressed that after solving 304 dailies this was the one that got me. Thanks.

**Comment by gereleth** [comment]
If a puzzle seems impossible you can always download it as json and import on the [custom puzzle page](https://hexapipes.vercel.app/custom). Then see what my solver makes of it =).
